extensions/notion/src/quick-capture.tsx

```diff
@@ -135,11 +177,33 @@ ${result?.content}
     getText();
   }, []);
 
+  function getQuicklink(): Quicklink {
+    const url = "raycast://extensions/notion/notion/quick-capture";
```

Comment:
question: Is there a better way to get this deep link value? Also, it looks like most of the Notion deeplinks use this value, but the deeplinks created from the "Create Database Page" command use the value "raycast://extensions/HenriChabrand/notion/create-database-page". Which value is correct?

Reply:
I believe both work. Both the `author` (HenriChabrand) and the `owner` (notion) field can be used to create the quicklink. I think the author version came from before `owner` was a valid field, and should be switched to use the `owner` version of the quicklink